src/worker.js

```diff
@@ -0,0 +1,44 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import dayjs from 'dayjs';
+import jp from 'jsonpath';
+
+/**
+ * This executed JS code inside a child process, so that the code
+ * has no access to the parent process and can be terminated on timeout etc
```

Comment:
It doesn't have access to parent process, but it would have the same privileges, no?

Reply:
I believe so, yes - from a Linux perspective. Node privileges do not yet propagate to child processes AFAIK.